compiler/record-minmax/src/HDF5Importer.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __RECORD_MINMAX_HDF5IMPORTER_H__
+#define __RECORD_MINMAX_HDF5IMPORTER_H__
+
+#include <luci_interpreter/core/Tensor.h>
+
+#include <H5Cpp.h>
+
+using DataType = luci_interpreter::DataType;
+
+namespace record_minmax
+{
+
+// HDF5Importer reads an input data saved in the hdf5 file in the given path
+// The hierarchy of the hdf5 file is as follows.
+// Group "/"
+//  > Group "value"
+//    > Group <record_idx>
+//      > Dataset <input_idx>
```

Comment:
When converting tf model to tflite model, users can specify the order of input nodes. Circle preserves the order (CircleInput has `index`). I think we can use the index instead of the name, under the assumption that the input order is preserved.

In fact, this is the same way as how runtime saves a record as hdf5.

Reply:
I see, thanks